perllib/FixMyStreet/Cobrand/Default.pm

```diff
@@ -713,6 +713,7 @@ sub available_permissions {
             planned_reports => _("Manage shortlist"),
             contribute_as_another_user => _("Create reports/updates on a user's behalf"),
             contribute_as_body => _("Create reports/updates as the council"),
+            view_body_contribute_details => _("See user detail for reports created as the council")
```

Comment:
Please do stick a comma on the last entry, so that the diff of any future new entry (such as this one) is a little easier :)